### PR TITLE
lokoctl installation document

### PR DESCRIPTION
**Pull request by ipochi**
Signed-off-by: Imran Pochi <imran@kinvolk.io>

### CODE REVIEW

File: docs/installer/lokoctl.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Once finished, `lokoctl` binary should be available in `$GOPATH/bin`.
+
+### Building from source
+
+Clone this repository and build the lokoctl binary:
+
+```bash
+git clone https://github.com/kinvolk/lokomotive
+cd lokomotive
+make
```
Comment (invidian) [inline docs/installer/lokoctl.md]: It should be noted, where the binary will be placed after running make.

File: docs/installer/lokoctl.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+lokoctl is a command line interface for Lokomotive, Kinvolk's open-source Kubernetes distribution
+which includes installers for various platforms and Lokomotive components.
+
+## Installation
+
+### Download official releases
+
+To install, find the [appropriate
+package](https://github.com/kinvolk/lokomotive/releases) for your system and
+download it. Lokoctl is packaged as a `tar.gz` archive.
```
Comment (iaguis) [inline docs/installer/lokoctl.md]:
```suggestion
download it. lokoctl is packaged as a `tar.gz` archive.
```

File: docs/installer/lokoctl.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+```console
+export PATH=$PATH:/path/to/lokoctl/binary
+```
+
+### Using 'go get'
+
+You can quickly get latest version of `lokoctl` by running following command:
+```
+go get github.com/kinvolk/lokomotive/cmd/lokoctl
```
Comment (surajssd) [inline docs/installer/lokoctl.md]: I know this is very trivial but saying that you should have golang installed on the system is important I think.

Comment (ipochi) [reply docs/installer/lokoctl.md]: I think this is implied.

File: docs/installer/lokoctl.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Clone this repository and build the lokoctl binary:
+
+```bash
+git clone https://github.com/kinvolk/lokomotive
+cd lokomotive
+make
+```
+
+The binary will be created in the same directory in which the `make` command was executed.
```
Comment (invidian) [inline docs/installer/lokoctl.md]: What if I do `make -C lokomotive` ? :wink: 

Maybe:
```suggestion
The binary will be created in the projects main directory.
```

**Pull request merged**